README.md

```diff
@@ -0,0 +1,42 @@
+# Endava test automation
+
+This repository contains simple Selenium tests for the endava.com website.
+
+
+# Examples :
+
+This project includes several Selenium examples:
```

Comment:
This test framework contains tests written for Endava website (https://www.endava.com/) and includes: